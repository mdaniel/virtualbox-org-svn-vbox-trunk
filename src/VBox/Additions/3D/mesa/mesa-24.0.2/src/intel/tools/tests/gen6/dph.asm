dph(8)          m4<1>.xF        g4<4>.xyzxF     g5<4>F          { align16 1Q };
dph.sat(8)      m4<1>F          g1<0>.xyzxF     g3<4>F          { align16 1Q };
dph(8)          m3<1>.xF        g5<4>.xyzxF     g1<0>F          { align16 NoDDClr 1Q };
dph(8)          m3<1>.yF        g5<4>.xyzxF     g1.4<0>F        { align16 NoDDClr,NoDDChk 1Q };
dph(8)          m3<1>.wF        g5<4>.xyzxF     g2.4<0>F        { align16 NoDDChk 1Q };
