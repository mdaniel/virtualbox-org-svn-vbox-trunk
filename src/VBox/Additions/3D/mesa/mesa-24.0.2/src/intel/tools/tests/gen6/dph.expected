55 01 60 00 be 77 81 20 84 00 62 00 a4 00 6e 00
55 01 60 80 be 77 8f 20 24 00 02 00 64 00 6e 00
55 05 60 00 be 77 61 20 a4 00 62 00 24 00 0e 00
55 0d 60 00 be 77 62 20 a4 00 62 00 34 00 0e 00
55 09 60 00 be 77 68 20 a4 00 62 00 54 00 0e 00
